src/solver/model_getters.jl

```diff
@@ -34,10 +34,12 @@ function get_sfc_temperature_form(parsed_args)
     end
 end
 
-function get_hyperdiffusion_model(parsed_args, ::Type{FT}) where {FT}
+function get_hyperdiffusion_model(parsed_args, spaces, ::Type{FT}) where {FT}
     hyperdiff_name = parsed_args["hyperdiff"]
-    κ₄_vorticity = FT(parsed_args["kappa_4_vorticity"])
-    κ₄_tracer = FT(parsed_args["kappa_4_tracer"])
+    h_space = Spaces.horizontal_space(spaces.center_space)
+    h_length_scale = Spaces.node_horizontal_length_scale(h_space)
```

Comment:
I suppose this is the mean nodal distance. It might be nice to document here what it is.

Reply:
Yes, it is the mean nodal distance.